tests/pages/system/overview/ClusterDetails.js

```diff
@@ -0,0 +1,42 @@
+describe("Cluster Details", function() {
```

Comment:
there has to be a `-cy` suffix in the filename for the "runner" script.

Reply:
just saw that I missed this on master 😢 these tests actually never run currently…
=> https://github.com/dcos/dcos-ui/pull/2904

Reply:
Thanks for catching it, fixed!